core/benchmarks/ZStandardBenchmark-jdk11-results.txt

```diff
@@ -6,22 +6,22 @@ OpenJDK 64-Bit Server VM 11.0.10+9-Ubuntu-0ubuntu1.18.04 on Linux 4.15.0-1044-aw
 Intel(R) Xeon(R) CPU E5-2670 v2 @ 2.50GHz
 Benchmark ZStandardCompressionCodec:                    Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns)   Relative
 --------------------------------------------------------------------------------------------------------------------------------------
-Compression 10000 times at level 1 without buffer pool           1165           1255         128          0.0      116483.1       1.0X
-Compression 10000 times at level 2 without buffer pool            932            981          42          0.0       93225.9       1.2X
-Compression 10000 times at level 3 without buffer pool           1176           1216          55          0.0      117634.8       1.0X
-Compression 10000 times at level 1 with buffer pool               415            424           5          0.0       41470.4       2.8X
-Compression 10000 times at level 2 with buffer pool               493            503           7          0.0       49295.1       2.4X
-Compression 10000 times at level 3 with buffer pool               754            759           6          0.0       75403.9       1.5X
+Compression 10000 times at level 1 without buffer pool            800            801           1          0.0       80017.0       1.0X
```

Comment:
What's the reason of these performance gains? The version upgrade or `ZstdInputStreamNoFinalizer`?

Reply:
It's purely `ZstdInputStreamNoFinalizer` and `ZstdOutputStreamNoFinalizer`.
`1.4.8-7` is only making it `public`.
- https://github.com/luben/zstd-jni/commit/57d53a09d247bcdb581d35dfec23a20b0661b86a